content/md/user-management/troubleshoot-the-sso.md

```diff
@@ -0,0 +1,58 @@
+---
+id: troubleshoot-the-sso
+themes: user-management
+title: How to **troubleshoot** the Single Sign-On
+popular: false
+ee-only: true
+related: configure-saml-sso
+priority: low
+---
+
+# Overview
+
+Single Sign-On is a powerful tool for productivity but sometimes, as multiple applications are concerned, it can be a hassle to troubleshoot when something goes wrong with it.
+
+Here are a few tips to help you understand what is going on between your `IdP` and your `SP`.
+
+# Make sure the IdP configuration is OK
+
+First, make sure that the configuration you have in your Identity Provider is consistent with the information from your SP:
+* the `Entity ID` of the SP
+* the `ACS URL`
+
+Make sure that there is no typo or missing characters.
```

Comment:
should we prioritise actions like 
1/ Make sure that there is no typo or missing characters.
2/ Ensure that the claims....
3/ Check if users are well created

Check this page for the format of lists: https://github.com/akeneo/pim-helpcenter/edit/master/content/md/productivity/projects-notifications.md